core/domain/email_manager_test.py

```diff
@@ -2422,12 +2422,13 @@ def test_that_correct_email_is_sent(self):
         email_text_body = 'Dummy email body.\n'
 
         with self.can_send_emails_ctx:
-            # pylint: disable=protected-access
-            email_manager._send_bulk_mail(
-                self.recipient_ids, self.sender_id,
-                feconf.BULK_EMAIL_INTENT_MARKETING, email_subject,
-                email_html_body, self.SENDER_EMAIL, self.SENDER_USERNAME)
-            # pylint: enable=protected-access
+            email_manager.send_user_query_email(
```

Comment:
Hi @brianlinUM, could you try to use the `send_user_query_email` method without adding extra parameters? We should avoid passing parameters like "test_only" to the actual method.

Reply:
Hi @kevintab95, thank you for your feedback! I made some changes so that the test calls send_user_query_email directly ( I found out it has the same functionality as the private method). I also reverted send_user_query_email to its original state. I forgot to fetch/merge from upstream/develop before making changes, so commit 509e936 is the one in which I made changes. Would this be a problem?